src/Umbraco.Web.BackOffice/Controllers/MemberController.cs

```diff
@@ -288,172 +310,275 @@ public ActionResult<MemberDisplay> GetEmpty(string contentTypeAlias = null)
+        /// <summary>
+        /// Create a member from the supplied member content data
+        /// 
+        /// All member password processing and creation is done via the identity manager
+        /// </summary>
+        /// <param name="contentItem">Member content data</param>
+        /// <returns>The identity result of the created member</returns>
+        private async Task<ActionResult<bool>> CreateMemberAsync(MemberSave contentItem)
         {
-            throw new NotImplementedException("Members have not been migrated to netcore");
-
-            // TODO: all member password processing and creation needs to be done with a new aspnet identity MemberUserManager that hasn't been created yet.
-
-            //var memberType = _memberTypeService.Get(contentItem.ContentTypeAlias);
-            //if (memberType == null)
-            //    throw new InvalidOperationException($"No member type found with alias {contentItem.ContentTypeAlias}");
-            //var member = new Member(contentItem.Name, contentItem.Email, contentItem.Username, memberType, true)
-            //{
-            //    CreatorId = _backofficeSecurityAccessor.BackofficeSecurity.CurrentUser.Id,
-            //    RawPasswordValue = _passwordSecurity.HashPasswordForStorage(contentItem.Password.NewPassword),
-            //    Comments = contentItem.Comments,
-            //    IsApproved = contentItem.IsApproved
-            //};
-
-            //return member;
+            IMemberType memberType = _memberTypeService.Get(contentItem.ContentTypeAlias);
+            if (memberType == null)
+            {
+                throw new InvalidOperationException($"No member type found with alias {contentItem.ContentTypeAlias}");
+            }
+
+            var identityMember = MembersIdentityUser.CreateNew(
+                contentItem.Username,
+                contentItem.Email,
+                memberType.Alias,
+                contentItem.Name);
+
+            IdentityResult created = await _memberManager.CreateAsync(identityMember, contentItem.Password.NewPassword);
+
+            if (created.Succeeded == false)
+            {
+                return new ValidationErrorResult(created.Errors.ToErrorMessage());
+            }
+
+            // now re-look up the member, which will now exist
+            IMember member = _memberService.GetByEmail(contentItem.Email);
+
+            // map the save info over onto the user
+            member = _umbracoMapper.Map<MemberSave, IMember>(contentItem, member);
+
+            int creatorId = _backOfficeSecurityAccessor.BackOfficeSecurity.CurrentUser.Id;
+            member.CreatorId = creatorId;
+
+            // assign the mapped property values that are not part of the identity properties
+            string[] builtInAliases = ConventionsHelper.GetStandardPropertyTypeStubs(_shortStringHelper).Select(x => x.Key).ToArray();
+            foreach (ContentPropertyBasic property in contentItem.Properties)
```

Comment:
@Shazwazza I added this because the customer member type properties weren't being saved. I've found a lot of ways this can be done around PersistedContent etc, but I went with this - best to have another set of eyes on it.

Also I saw the key being set again elsewhere, but I'm not sure if this is necessary here or not.

Reply:
I'll have to check this out with the debugger, not really sure off the top of my head.